### DISCUSSION TITLE
官网宣传图问题

### DISCUSSION THREAD

**Discussion by SekiBetu** [cloudreve/cloudreve#1322] [General]
![image](https://user-images.githubusercontent.com/38283893/170856321-750ed45d-06fc-4bb7-acc1-687d8502ce23.png)

我觉得还是谨慎点把文件名打码了，这个明显盗版电影资源

**Comment by HFO4** [comment]
Thanks for bring it up! I've replace the image with public domain torrents.

**Comment by elarbor** [comment]
好难打开啊
![image](https://github.com/user-attachments/assets/9e24cc44-11b1-44ae-ba00-c278ab023de6)

**Reply by elarbor** [reply]
加载好久了
